stdlib/arg.mli

```diff
@@ -23,9 +23,13 @@
     An option is a keyword alone or followed by an argument.
     The types of keywords are: [Unit], [Bool], [Set], [Clear],
     [String], [Set_string], [Int], [Set_int], [Float], [Set_float],
-    [Tuple], [Symbol], and [Rest].
-    [Unit], [Set] and [Clear] keywords take no argument. A [Rest]
-    keyword takes the remaining of the command line as arguments.
+    [Tuple], [Symbol], [Rest], [Rest_all] and [Expand].
+
+    [Unit], [Set] and [Clear] keywords take no argument.
+
+    A [Rest] or [Rest_all] keyword takes the remaining of the command line
```

Comment:
I followed your recommendation, but I made small changes to the first paragraph/sentence (I found it confusing), and moved them to the end of the documentation paragraph, because I think the examples are more important.

Reply:
I like it!